cpp/src/parquet/encryption/encryption.h

```diff
@@ -327,19 +322,19 @@ class PARQUET_EXPORT FileDecryptionProperties {
   }
 
  private:
-  std::string footer_key_;
+  encryption::SecureString footer_key_;
   std::string aad_prefix_;
   std::shared_ptr<AADPrefixVerifier> aad_prefix_verifier_;
 
-  const std::string empty_string_ = "";
+  const encryption::SecureString empty_string_ = encryption::SecureString("");
```

Comment:
Is `empty_string_` useful at all? It looks rather dubious.

Reply:
It is, method `FileDecryptionProperties::column_key` returns an empty string as the encryption key when the key id is not found. It should rather throw a meaningful `key not found` exception. I may fix that while I am touching this code anyway.

Reply:
Still needed, improved name.